Test/Web/FunctionalTests/TestApps/WebAppFW45/packages.config

```diff
@@ -1,11 +1,13 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.ApplicationInsights" version="2.4.0-beta1-build07987" targetFramework="net451" />
   <package id="Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel" version="2.4.0-beta1-build07987" targetFramework="net451" />
+  <package id="Microsoft.AspNet.Diagnostics" version="1.0.0-beta-10405-01" targetFramework="net451" />
```

Comment:
Why do you think it should be aligned? It's an HTTP module, it could work with any ASP.NET version